vividus-plugin-html/src/main/java/org/vividus/util/HtmlUtils.java

```diff
@@ -28,7 +28,12 @@ private HtmlUtils()
 
     public static Elements getElements(String html, String cssSelector)
     {
-        Document document = Jsoup.parse(html);
+        return getElements("", html, cssSelector);
```

Comment:
how is empty base URI treated by Jsoup?

Reply:
the base URI is used to resolve relative urls, if it's empty then relative urls are returned as is (default behaviour)

Reply:
org.jsoup.Jsoup.parse(String html) ?

Reply:
this method reuses parsing and selecting logic in the method below

Reply:
<img width="179" alt="image" src="https://user-images.githubusercontent.com/30923336/130419767-368ec7ac-7876-493a-ab47-9945bde2e06b.png">